federation/apis/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Generic API Server",
-   "version": "v1.7.0"
+   "version": "v1.6.0"
```

Comment:
Same here - 
I am not sure of this change and it looks bit strange to me. I didn't make any manual change, while running "git commit" kube gave an error message:
"ERROR!
Federation OpenAPI spec needs to be updated.
To regenerate the spec, run:
  hack/update-federation-openapi-spec.sh" and I guess this change in version is probably an output of "hack/update-federation-openapi-spec.sh"  Thanks!